packages/devextreme/scss/widgets/fluent/htmlEditor/_sizes.scss

```diff
@@ -9,6 +9,8 @@ $fluent-html-editor-horizontal-padding: null !default;
 $fluent-html-editor-add-image-dialog-fileuploader-padding: null !default;
 $fluent-html-editor-toolbar-menu-separator-margin: 4px !default;
 $fluent-html-editor-fileuploader-input-wrapper-border-radius: 8px !default;
+$fluent-html-editor-fileuploader-input-wrapper-border-size: 1.5px !default;
```

Comment:
I don't think that using not integer values in this case is justified. It will better to change color of the border for better contrast.

Reply:
This was discussed with the designer.